buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -1,5 +1,28 @@
 {
   "build": [
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": false,
```

Comment:
We should understand what the difference between "alwaysRun"= false and condition always() and be sure the don't conflict with each other.